### PR TITLE
font-awesome icons where possible. Ext buttons look and behave in line with Ext tab headers. Mouse over highlighting of buttons.

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

File: web-app/css/general.css [hunk 1]
```diff
@@ -1118,3 +1117,8 @@ div.tool-plus-disabled {
     opacity: 0.8;
     font-family: FontAwesome! important;
 }
+
+.fa-lg.fa-angle-double-right,
```
Comment (danfruehauf) [inline web-app/css/general.css]: I think be consistent here and have them both on the same line.

File: web-app/js/portal/search/GeoSelectionPanel.js [hunk 2]
```diff
@@ -33,30 +33,37 @@ Portal.search.GeoSelectionPanel = Ext.extend(Ext.Panel, {
...
+            height: 280,
+            width: 300
         });
 
         var config = Ext.apply({
             layout:'form',
             cls:'search-filter-panel filter-selection-panel',
+            defaults: {
+                style: {
+                    margin: '2px'
```
Comment (danfruehauf) [inline web-app/js/portal/search/GeoSelectionPanel.js]: Can this be moved to CSS potentially?

File: web-app/js/portal/search/GeoSelectionPanel.js [hunk 3]
```diff
@@ -33,30 +33,37 @@ Portal.search.GeoSelectionPanel = Ext.extend(Ext.Panel, {
         this.facetMap = new Portal.search.FacetMapPanel({
             initialBbox: Portal.app.appConfig.portal.initialBbox,
             mainMap: cfg.mapPanel,
-            height: 250,
-            width: 250
+            height: 280,
+            width: 300
```
Comment (danfruehauf) [inline web-app/js/portal/search/GeoSelectionPanel.js]: Any reason this gets "fatter"?

File: web-app/js/portal/search/GeoSelectionPanel.js (hunk 2 above)
Comment (pmbohm) [inline web-app/js/portal/search/GeoSelectionPanel.js]: This is the better way to go in this particular case I think. I prefer to have less CSS rules created as they sometimes have adverse outcomes. Very specific CSS rules also go stale as you saw not too long ago. Also using Id's are not the best way to go as we latter shuffle code

File: web-app/js/portal/search/GeoSelectionPanel.js (hunk 3 above)
Comment (pmbohm) [inline web-app/js/portal/search/GeoSelectionPanel.js]: yes, it now fits the space better

File: web-app/css/general.css (hunk 1 above)
Comment (pmbohm) [inline web-app/css/general.css]: What consistency am i following? This is readable to me

**Comment by pmbohm** [comment]
@danfruehauf review comments addressed.

File: web-app/css/general.css (hunk 1 above)
Comment (danfruehauf) [inline web-app/css/general.css]: All the rest of the times you have both in the same line. I don't think it's critical, it's just inconsistent.

**Pull request merged**